pkg/features/kube_features.go

```diff
@@ -856,6 +856,17 @@ const (
 	//
 	// Enables specifying resources at pod-level.
 	PodLevelResources featuregate.Feature = "PodLevelResources"
+
+	// owner: @ffromani
+	// beta: v1.33
+	//
+	// Disable CPU Quota for containers which have exclusive CPUs allocated.
+	// Disables pod-Level CPU Quota for pods containing at least one container with exclusive CPUs allocated
+	// Exclusive CPUs for a container (init, application, sidecar) are allocated when:
```

Comment:
ok, we declare we want to do also for init containers, which I'm not sure it's worth it but should not be wrong either, so I'm neutral about it.